newsroom/push.py

```diff
@@ -725,14 +725,19 @@ def notify_agenda_topic_matches(item, users_dict):
 
 def send_topic_notification_emails(item, topics, topic_matches, users):
     for topic in topics:
-        user = users.get(str(topic['user']))
-        if topic['_id'] in topic_matches and user and user.get('receive_email'):
-            send_new_item_notification_email(
-                user,
-                topic['label'],
-                item=item,
-                section=topic.get('topic_type') or 'wire'
-            )
+        if topic['_id'] not in topic_matches:
+            continue
+
+        for user_id in topic.get('subscribers') or []:
+            user = users.get(str(user_id))
+
+            if user and user.get('receive_email'):
+                send_new_item_notification_email(
```

Comment:
would be good to group that somehow, probably per item so we would send 1 email per 1 item listing the matching topics instead of 1 email for each topic